javalib/src/main/scala/java/net/Inet6Address.scala

```diff
@@ -45,7 +45,8 @@ final class Inet6Address private (
   override def hashCode(): Int = {
     var res = 1
     res = 31 * res + ju.Arrays.hashCode(ipAddress)
```

Comment:
Ipaddress can be null too

Reply:
Good catch!  Thank you.

It is not evident to a reviewer and certainly not to me when I wrote it, but `ju.Arrays.hashCode(null)`
always returns 0. That is both robust sensible.  

I added a comment to remind myself and others who try to verify that method in the future.

Dratted nulls!